themes/jquery/footer.php

```diff
@@ -36,7 +36,12 @@
 	</div>
 </footer>
 
-<?php wp_footer(); ?>
+<?php
+	wp_footer();
+
+	if ( function_exists( 'footer_append' ) )
+		footer_append();
```

Comment:
I wasn't sure about this approach, so I'd be interested in your feedback and possible alternative. It does allow other themes to follow suit, but feels a little hacky at the same time.